src/50_Samples_%26_Templates/Quiz.html

```diff
@@ -0,0 +1,161 @@
+    line-height: 36px;
+    vertical-align: middle;
+    color: #fff;
+    background-color: #eb407a;
+    box-shadow: 0 2px 2px 0 rgba(0,0,0,.14),0 3px 1px -2px rgba(0,0,0,.2),0 1px 5px 0 rgba(0,0,0,.12);
+  }
+
+</style>
+</head>
+<body>
+  <!-- #### Arithmetical Quiz -->
+  <!--
+  We use [amp-form](https://github.com/ampproject/amphtml/blob/master/extensions/amp-form/amp-form.md) to implement a quiz.
+
+  Questions are shown one at the time using [amp-bind](https://github.com/ampproject/amphtml/blob/master/extensions/amp-bind/amp-bind.md).
+
+  The global variable `currentQuestion` keeps the current question index and it's used to show the question using the
+  following binding: `[class]="currentQuestion == 1 ? 'show' : 'hide'"`.
+
+  The global variable `quizDone` is initialized to true after the last question is answered and it's used to show the submit button using the
+  following binding: `[class]="quizDone == true ? 'show' : 'hide'"`
+
+-->
+<form method="post" action-xhr="/samples_templates/quiz/submit" target="_blank"
+  id="quiz-form">
+  <h3>Answer the following mathematical questions:</h3>
+  <div class="show" [class]="currentQuestion == 1 ? 'show' : 'hide'">
+    <h1>2+2</h1>
+    <amp-selector name="expression1" layout="container">
+      <ul class="answer-container" on="tap:AMP.setState(currentQuestion=2)" >
+        <li option="4">4</li>
+        <li option="5">5</li>
+        <li option="8">8</li>
+        <li option="0">0</li>
+      </ul>
+    </amp-selector>
+  </div>
+  <div class="hide" [class]="currentQuestion == 2 ? 'show' : 'hide'">
+    <h1>4:2</h1>
+    <amp-selector name="expression2" layout="container">
+      <ul class="answer-container" on="tap:AMP.setState(currentQuestion=3)" >
+        <li option="4">4</li>
+        <li option="2">2</li>
+        <li option="8">8</li>
+        <li option="0">0</li>
+      </ul>
+    </amp-selector>
+  </div>
+  <div class="hide" [class]="currentQuestion == 3 ? 'show' : 'hide'">
+    <h1>4*4</h1>
+    <amp-selector name="expression3" layout="container">
+      <ul class="answer-container" on="tap:AMP.setState(currentQuestion=4)" >
+        <li option="24">24</li>
+        <li option="16">16</li>
+        <li option="8">8</li>
+        <li option="0">0</li>
+      </ul>
+    </amp-selector>
+  </div>
+  <div class="hide" [class]="currentQuestion == 4 ? 'show' : 'hide'">
+    <h1>1*1</h1>
+    <amp-selector name="expression4" layout="container">
+      <ul class="answer-container" on="tap:quiz-form.submit">
+        <li option="1">1</li>
+        <li option="-1">-1</li>
+        <li option="8">8</li>
+        <li option="0">0</li>
+      </ul>
+    </amp-selector>
+    <div submit-success>
+      <template type="amp-mustache">
+        Success! You answered all the right questions!
+      </template>
+    </div>
+    <div submit-error>
+      <template type="amp-mustache">
+          <a class="button button-primary" href="/samples_templates/quiz">Try again</a>
```

Comment:
Discussing this on [amphtml issue](https://github.com/ampproject/amphtml/issues/7688)